.gitignore

```diff
@@ -1,4 +0,0 @@
-.DS_Store
```

Comment:
Task for you to learn about .gitignore. This file is great to have in any git project.
Also please see below - you've added binary files to the project. Please remove them.